legal-api/src/legal_api/core/filing.py

```diff
@@ -409,9 +409,16 @@ def get_document_list(business, filing, request) -> Optional[dict]:
                                                    'filing_id': filing.id,
                                                    'legal_filing_name': None})
 
-        documents = {'documents': {
-                     'receipt': f'{base_url}{doc_url}/receipt'
-                     }}
+        documents = {'documents': {}}
+
+        has_payment_rules = [
+            filing.storage,
+            filing.storage.payment_status_code,
+            filing.storage.payment_status_code == 'COMPLETED'
```

Comment:
I think you want to add `filing.storage.payment_completion_date` to `has_payment_rules` too.  When the payment queue message is officially processed by entity-pay queue listener, `payment_completion_date` is populated.

Reply:
Is there scenarios where there could be a `COMPLETED` state without a `payment_completion_date` or vice versa?
Are they both required indicators that a payment has been made, or is one or another a better indicator?
Can definitely add it in, just trying to deduce the best option.

Reply:
yeah, i think just using `payment_completion_date` would be ideal.  Let me dig through the data a bit.  @thorwolpert any thoughts?  should just checking with `payment_completion_date` be good enough?

Reply:
I'd be careful on the list set up, as it relies on those pieces being not None anyways, unlike an if evaluation, so they can be dropped in the list setup.


```status == 'COMPLETED'``` without payment for filings loaded from COLIN
and we'll need to figure out what to do for data loading, as we have the historical payment info there ... 
In our current state though, I think @argush3 is right that we only need to check the ```payment_completion_date``` for filings that will a receipt. 
For historical filings, where it's not paper_only and COMPLETED, the filings can be shown, but **not** the receipt

Reply:
Awesome! Ok thanks, will make these adjustments 👍

Reply:
Just in testing against DEV data, i've noticed a lot of filings that have a `payment_completion_date` value and the `payment_status_code` is `None`. 
Is this a scenario we might expect in a real environment. I ask, because if we use this DATE we are still presenting receipt links for the user but they all result in 404.